lib/v2/decoder.js

```diff
@@ -184,13 +186,15 @@ proto.readLong = function () {
     var b0 = this.byteBuffer.get();
     return ((code - 0x3c) << 16) + (b1 << 8) + b0;
   }
-  // ::= x77 b3 b2 b1 b0       # 32-bit integer cast to long
-  if (code === 0x77) {
-    // Longs between which fit into 32-bits are encoded in five octets with the leading byte x59.
-    // value = (b3 << 24) + (b2 << 16) + (b1 << 8) + b0
+  // // ::= x77 b3 b2 b1 b0       # 32-bit integer cast to long
+  // if (code === 0x77) {
+  //   // Longs between which fit into 32-bits are encoded in five octets with the leading byte x59.
+  //   // value = (b3 << 24) + (b2 << 16) + (b1 << 8) + b0
+  //   return this.byteBuffer.getInt32();
+  // }
+  if (code === 0x59) {
```

Comment:
long       ::= 'L' b7 b6 b5 b4 b3 b2 b1 b0
           ::= [xd8-xef]             # -x08 to x0f
           ::= [xf0-xff] b0          # -x800 to x7ff
           ::= [x38-x3f] b1 b0       # -x40000 to x3ffff
           ::= x59 b3 b2 b1 b0       # 32-bit integer cast to long

http://hessian.caucho.com/doc/hessian-serialization.html#anchor3